packages/flutter/test/widgets/scrollable_restoration_test.dart

```diff
@@ -14,13 +14,11 @@ void main() {
           restorationId: 'list',
           cacheExtent: 0,
           slivers: <Widget>[
-            SliverList(
-              delegate: SliverChildListDelegate(
-                List<Widget>.generate(
-                  50,
-                  (int index) => SizedBox(height: 50, child: Text('Tile $index')),
-                ),
-              ),
+            SliverList.builder(
```

Comment:
I also switched to `SliverList.builder` here, let me know if you'd prefer I use `SliverList.list` here instead.